packages/components/src/components/Flyout/Flyout.stories.tsx

```diff
@@ -264,6 +264,39 @@ export const MobileView: Story = {
     },
 };
 
+export const MobileViewBottom: Story = {
```

Comment:
There is already a mobile view https://fondue-components.frontify.com/?path=/story/current_components-flyout--mobile-view&globals=viewport:mobile

Reply:
the button is at the bottom, not strictly necessary it's just to show what happens when we position the trigger differently